app/src/comic/admin.py

```
@@ -40,6 +41,11 @@ class SeriesAdmin(admin.ModelAdmin):
     def latest_volume_tw_display(self, obj):
         return obj.latest_volume_tw
 
+    @admin.action(description="手動發送本週新書通知 (Celery)")
+    def trigger_weekly_digest(self, request, queryset):
```

Ruff (0.14.11)

Unused method argument: `queryset`

(ARG002)